devops/lib/utils.py

```diff
@@ -183,7 +183,10 @@ def _nest(_doc, _overrides, _path):
                     res[key] = doc[key]
                 elif type(overrides[key]) in (str, int, bool, float, complex):
                     # Simply overridden values
-                    res[key] = _basevalue_to_value(overrides[key], path)
+                    if type(_basevalue_to_value(overrides[key], path)) == int:
```

Comment:
What about boolean values? such as `allowPrivilegeEscalation: true` which is converted to `allowPrivilegeEscalation: 'True'`. This also fails validation.